drivers/dma-buf/sync_file.c

```diff
@@ -488,5 +488,7 @@ static const struct file_operations sync_file_fops = {
 	.compat_ioctl = compat_ptr_ioctl,
 #elif defined(__FreeBSD__)
 	.compat_ioctl = sync_file_ioctl,
+	.read = NULL,
```

Comment:
Why all these NULL initializers exist? Static variables are initialized to 0 by default by C specs